modules/ROOT/pages/inline-css.adoc

```diff
@@ -37,55 +37,151 @@ tinymce.init({
-*** The user can also manually insert a stylesheet link in the <head> of the page.
-* A CSS stylesheet that is specified in the `<body>` of the page i.e. `<link rel="stylesheet" href="styles.css">`
-** This is not recommended by https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link[MDN], but some browsers do allow it
-* A `<style>` in the `<head>` of the page i.e. `<style>p { color: red; }</style>`
-** To add a `<style>` in {productname} is by using the `content_style` option which automatically creates the `<style>` tag in the <head> of the document
-** The user can also manually put a `<style>` tag in the <head> of the page.
-* A `<style>` in the `<body>` of the page
-** As noted in the `Classic iframe` section, this should not be done, but some browsers will still allow it.
-* A `style` attribute on an {productname} element within the content i.e. inline CSS
+When a {productname} editor in classic mode is contained within a shadow root, CSS applied to the editor’s content will work as expected if the editor can also safely assume it is not operating in a shadow root.
 
-=== Shadow DOM is supported but **not** officially, with some notable limitations
 
-Shadow DOM is **not** officially supported by {productname}, but the feature will work when an editor running in `inline` or `iframe` is contained within a shadow root. 
+==== Inline mode
 
-When an `iframe` mode editor is contained with a shadow root, the editor content can be styled the same way, where the expected behavior for the `feature` is that it should not have to consider if the editor is in a shadow root. When an `inline` editor is contained within a shadow root, the stylesheet (`<link>` tags) and `<style>` tags are added directly as children of the shadow root and do not need to be contained within a `<head>` tag. 
+When a {productname} editor in inline mode is contained within a shadow root, tags pointing to CSS declarations — either `<link>` tags pointing to stylesheet files or `<style>` tags containing specific CSS examples — are added as children of the shadow root. These tags do not need to be contained with the `<head>` tag.
 
-When in `inline mode`, the feature checks if the editor is inside a shadow root to ensure it does not try to get `<styles>` that are outside the shadow root.
+As well, when {productname} is working in inline mode, {pluginame} checks if the editor is inside a shadow root. If it finds it is, {pluginame} does not look for CSS declarations outside the shadow root.
 
-When an editor is in a shadow root, the feature should still support getting the stylesheets specified via the `content_css` option as well as the CSS styles specified via  the`content_style` option.
 
 // * some pseudo-class selectors. See the specification for more details
 // (https://notion.so/tinycloud/Technical-specification-7c16d4522d1241ba9e7364880eab3e51#94b1d13ff444492bbfce789f7876f39c)
```

Comment:
Might be good to remove this line, the notion is intended to be internal only

Reply:
> Might be good to remove this line, the notion is intended to be internal only

Removed.